src/components/frame/v5/pages/UserPreferencesPage/hooks.tsx

```diff
@@ -93,3 +94,24 @@ export const useUserPreferencesPage = () => {
     errors,
   };
 };
+
+export const useFullScreenMode = () => {
+  const [isFullScreenMode, setIsFullScreenMode] = useState(false);
+
+  useLayoutEffect(() => {
```

Comment:
could we just pass this into the initial state value?

Reply:
@bassgeta Accessing `localStorage` is a side effect and thus belongs in an effect, see https://ui.dev/c/react/effects (around half way down they show this exact example)

Reply:
sure thing, it's probably not gonna cause issues since we're using `useLayoutEffect`